Tools/scripts/build_options.py

```diff
@@ -347,7 +347,8 @@ def __init__(self,
     Feature('Precision Landing', 'PrecLand', 'AC_PRECLAND_ENABLED', 'Enable Precision Landing support', 0, None),
     Feature('Precision Landing', 'PrecLand - Companion', 'AC_PRECLAND_COMPANION_ENABLED', 'Enable Companion-Supported Precision Landing support', 0, "PrecLand"),  # noqa
     Feature('Precision Landing', 'PrecLand - IRLock', 'AC_PRECLAND_IRLOCK_ENABLED', 'Enable IRLock-Supported Precision Landing support', 0, "PrecLand"),  # noqa
-
+    Feature('Precision Landing', 'PrecLand - SITL_GAZEBO', 'AC_PRECLAND_SITL_GAZEBO_ENABLED', 'Enable SITL-GAZEBO Precision Landing support', 0, "PrecLand"),
```

Comment:
I would imagine that trying to compile these into stm32-based boards won't work at the moment, and probably wouldn't be useful if they did compile in :-)

Reply:
Yes, That's totally makes sense, if we are doing it for stm32 then we wont be needing the sitl builds.